pkg/session/serve.go

```diff
@@ -48,6 +48,9 @@ type Request struct {
 	// ComponentName is the name of the component to query or deregister.
 	ComponentName string `json:"component_name,omitempty"`
 
+	// TagName is the tag of the component to trigger check.
+	TagName string `json:"tag_name,omitempty"`
```

Comment:
How to support the case where someone wants to trigger multiple components via a tag?

Reply:
@eahydra User can configure tags via this field

https://github.com/leptonai/gpud/blob/2bb317e8c0002549be6cbde1aa39a59f94df85cc/pkg/custom-plugins/types.go#L73-L77

And send the request with the matching tag

Reply:
I mean, does requests support triggering multiple components selected by tags for checking at once?

Reply:
Yes